docs/content/Deployment/Overview.mdx

```diff
@@ -57,14 +57,14 @@ requests between multiple API instances.
 
 The [Cube Docker image][dh-cubejs] is used for API Instance.
 
-API instance needs to be configured via environment variables, cube.js file and
-has access to the data schema files.
+API instance needs to be configured via environment variables, the `cube.js`
+configuration file and has access to the data model files.
```

Comment:
Sounds weird. Who/what has access to the data model files? Maybe you can quickly untangle this now 🙂